views/account_pages/account.ejs

```diff
@@ -93,20 +93,23 @@
    <p>If you represent a business you can <a class="govuk-link" href="/api/user/upgrade-account">register for the legalisation premium service</a></p>
 </div>
 <%}%>
-<h1 class="heading-xlarge" style="margin-bottom: 25px">Saved payment details</h1>
-<div class="column-two-thirds">
-    <p>You can save your card details so that you don't have to type them in each time you apply.</p>
-    <p>Follow these steps to save a card when you make an application:</p>
-    <ul>
-        <li>enter your card details</li>
-        <li>ensure the 'Remember these details' box is ticked when you are asked to review and complete your payment - this will save the card</li>
-        <li>next time you make an application, you'll see your saved card at the top of the payment page and can use it to pay</li>
-        <li>you can delete saved cards or add additional cards each time you go to pay</li>
-    </ul>
-    <div class="panel panel-border-narrow" style="margin-top: 5px">
-        For security reasons, any cards you save are only displayed on the Barclaycard Smartpay payment pages
-    </div>
+<!--Commenting this bit out as gov pay does not currently save card details-->
 
-</div>
+
+<!--<h1 class="heading-xlarge" style="margin-bottom: 25px">Saved payment details</h1>-->
+<!--<div class="column-two-thirds">-->
+<!--    <p>You can save your card details so that you don't have to type them in each time you apply.</p>-->
+<!--    <p>Follow these steps to save a card when you make an application:</p>-->
+<!--    <ul>-->
+<!--        <li>enter your card details</li>-->
+<!--        <li>ensure the 'Remember these details' box is ticked when you are asked to review and complete your payment - this will save the card</li>-->
+<!--        <li>next time you make an application, you'll see your saved card at the top of the payment page and can use it to pay</li>-->
+<!--        <li>you can delete saved cards or add additional cards each time you go to pay</li>-->
+<!--    </ul>-->
+<!--    <div class="panel panel-border-narrow" style="margin-top: 5px">-->
+<!--        For security reasons, any cards you save are only displayed on the Barclaycard Smartpay payment pages-->
+<!--    </div>-->
+
+<!--</div>-->
```

Comment:
I think we should just remove this instead of commenting it out. It would be cleaner, as if we never get the 'saved card' functionality, then this won't be lingering around forever.